Gemfile.lock

```diff
@@ -189,6 +189,8 @@ GEM
       jekyll (>= 3.0)
     kramdown (1.16.2)
     liquid (4.0.0)
+    liquid-md5 (0.0.3)
```

Comment:
Is this being used anywhere? I couldn't find it.

Reply:
No, it was going to be, but it won't work on github pages.